### ISSUE TITLE
Suggestion -  update Trash Cans with other dead players loot

### ISSUE THREAD

**Issue by wahoovisuals** [InZidiuZ/op-framework-issue-tracker#3324]
### Assurance

I have read the `README.md`

### Summary

After dying and pressing "Enter" sometimes our stuff gets stolen on the way to hospital. instead of completely losing everything maybe just teleport the missing items to random locations or trash cans. 

### Reason

so the items are still in circulation and other people can steal it if they come across it.

**Comment by coalaura** [comment]
no, i don't think its a good idea to hand off personal items to other people through dumpsters
